documentation/source/Introduction/pages/contributing.rst

```diff
@@ -9,6 +9,19 @@ If you intend to contribute significantly to NuRadioMC, please contact
 `@anelles <https://github.com/anelles>`_ and `@cg-laser <https://github.com/cg-laser>`_ on GitHub so that we are informed about ongoing
 activities. Both will also be able to provide commit access to the repository.
 
+Installing NuRadioMC for developers
+-----------------------------------
```

Comment:
Does that mean that this is optional?

Reply:
If you mean installing the pre-commit hook, the answer is yes; it is (for good reason) impossible to automatically install hooks which could in principle execute arbitrary code every time you invoke `git`. I have tried to word it as strongly as possible as I don't see any downsides and many upsides to everyone installing this hook, but there is (at least on github) no way to enforce this.

Reply:
To your other question: pre-commit install installs a git hook, which will be invoked every time you use `git commit`. I checked and if you do this from an environment without `pre-commit` installed the hook (and therefore the commit) will fail.

Reply:
but only for this repo right?

Reply:
so please go ahead and merge this thing